src/basic/Movable.h

```diff
@@ -26,25 +26,37 @@ class Movable : public virtual Entity {
 public:
     //for Marines and Zombies
     Movable(const int32_t id, const SDL_Rect& dest, const SDL_Rect& movementSize, const SDL_Rect& projectileSize,
-            const SDL_Rect& damageSize, const int vel) : Entity(id, dest, movementSize, projectileSize,
-            damageSize), velocity(vel), dx(0), dy(0), angle(0.0) {};
+        const SDL_Rect& damageSize, const int vel) : Entity(id, dest, movementSize, projectileSize,
+        damageSize), velocity(vel), dx(0), dy(0), angle(0.0) {};
 
     //for turrets
     Movable(const int32_t id, const SDL_Rect& dest, const SDL_Rect& movementSize, const SDL_Rect& projectileSize,
-            const SDL_Rect& damageSize, const SDL_Rect& pickupSize, const int vel) : Entity(id, dest, movementSize,
-            projectileSize, damageSize, pickupSize), velocity(vel), dx(0), dy(0), angle(0.0) {};
+        const SDL_Rect& damageSize, const SDL_Rect& pickupSize, const int vel) : Entity(id, dest, movementSize,
+        projectileSize, damageSize, pickupSize), velocity(vel), dx(0), dy(0), angle(0.0) {};
 
     virtual ~Movable() = default;
-
-    void move(float moveX, float moveY, CollisionHandler& ch); // Moves Marine
-    void setDX(float px); //set delta x coordinate
-    void setDY(float py); //set delta y coordinate
-    void setVelocity(int pvel); // set velocity of Marine movement
-    float getDX() const; // get delta x coordinate
-    float getDY() const; //get delta y coordinate
-    int getVelocity() const; // get velocity of Marine movement
-    void setAngle(const double a);//sets angle of sprite to
-    double getAngle() const; //returns sprites angle
+    // Moves Marine
+    void move(const float moveX, const float moveY, CollisionHandler& ch);
+    //set delta x coordinate
+    void setDX(float px) {dx = px;}
+    //set delta y coordinate
+    void setDY(float py) {dy = py;}
+    // set velocity of Marine movement
+    void setVelocity(int pvel) {velocity = pvel;}
```

Comment:
You're missing a const
Oh the travesty!
It's not 🌮 time
Please fix this spicy burrito